packages/internal/handle-resolver/src/atproto-doh-handle-resolver.ts

```diff
@@ -54,7 +54,7 @@ function dohResolveTxtFactory({
           ? await response.text()
           : `Failed to resolve ${hostname}`
         throw new TypeError(message)
-      } else if (contentType !== 'application/dns-json') {
+      } else if (contentType?.match(/application\/(dns-)json/i) == null) {
```

Comment:
Did you mean to make the `(dns-)` part optional ? Or is there no particular meaning behind the parenthesis there ?

Reply:
oh god, it definetly must be optional, that's half the point of my PR, sorry about that. I fixed it.